src/main/java/core/basesyntax/service/operations/OperationIncreaseHandler.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.service.operations;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.model.Fruit;
+import java.util.Optional;
+
+public class OperationIncreaseHandler implements OperationHandler {
+    private final FruitDao fruitDao;
+
+    public OperationIncreaseHandler(FruitDao fruitDao) {
+        this.fruitDao = fruitDao;
+    }
+
+    @Override
+    public int apply(int quantity, Fruit key) {
```

Comment:
do we have a check for amount, like, "-10"?

Reply:
Done!